rpc/src/module/chain.rs

```diff
@@ -81,11 +82,13 @@ impl ChainRpc for ChainRpcImpl {
     }
 
     fn get_header(&self, hash: H256) -> Result<Option<HeaderView>> {
-        Ok(self
-            .shared
-            .snapshot()
-            .get_block_header(&hash.pack())
-            .map(Into::into))
+        let snapshot = self.shared.snapshot();
+
+        if !snapshot.is_main_chain(&hash.pack()) {
+            return Ok(None);
```

Comment:
What if I wanna get an uncle-block? Add another RPC interface later?

Reply:
fork-block? `get_fork_block` is only for the test now.

Reply:
Yeah, that is what I want. Thanks.